react/components/Header/Summary.js

```diff
@@ -18,23 +18,24 @@ const OrderSummary = ({ data }) => {
   const longestDeliveryEstimate = estimateCalculator.getLatestSla(delivery)
   const longestPickUpEstimate = estimateCalculator.getLatestSla(pickup)
 
-  return (
-    <div className="flex justify-around mt7 mb9 flex-wrap">
-      <div className="w-40-l w-90">
+  return (delivery.length > 0 && pickup.length > 0) &&
+  (
+    <div className="flex justify-around mt7 mb9 flex-wrap w-80-ns w-90 center bb b--muted-4">
+      <div className="w-40-l w-90 mb8">
```

Comment:
Could this be updated to use `<PageBlock/>`?

Reply:
Yep! Just updated it :)